Amsterdam Internet Exchange/README.md

```diff
@@ -0,0 +1,7 @@
+# Amsterdam Internet Exchange
+
+This scripted connector scrapes data from the AMS-IX website for various locations, including Amsterdam, Bay Area, Caribbean, Chicago, Hong Kong, and Mumbai. It then sends the collected data to a Data API. 
```

Comment:
I would add also the URL https://www.ams-ix.net/ to be clear what website you refer to